src/change_stream.ts

```diff
@@ -277,7 +278,7 @@ export class ChangeStream extends EventEmitter {
   hasNext(callback?: Callback): Promise<void> | void {
     return maybePromise(callback, cb => {
       getCursor(this, (err, cursor) => {
-        if (err || !cursor) return cb(err); // failed to resume, raise an error
+        if (err || !cursor) return cb(err || NO_CURSOR_ERROR); // failed to resume, raise an error
```

Comment:
FWIW it looks like if you need to add `NO_CURSOR_ERROR` to each call to `getCursor` then the error should probably be returned from `getCursor`? (maybe per the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)))